framework/src/controller/application.js

```diff
@@ -78,6 +78,10 @@ module.exports = class Application {
 		constants = {},
 		config = { components: { logger: null } }
 	) {
+		if (typeof label === 'function') {
+			label = label.call();
+		}
+
 		if (!config.components.logger) {
 			config.components.logger = {
 				filename: `~/.lisk/${label}/lisk.log`,
```

Comment:
Also, need to align the folder naming and contents with `lisk-commander` as from `v1.0` we are using `lisk` command with default folder `.lisk`

Reply:
This change will not happen to core as we decide to keep the project root. The code you mentioned above will be fixed in other PR, which specifies default values for each component configuration.